app/Http/Controllers/SyncController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AtlasVG\Http\Controllers;
+
+use AtlasVG\Http\Controllers\Controller;
+use AtlasVG\TokenStore\TokenCache;
+use Microsoft\Graph\Graph;
+use Microsoft\Graph\Model;
+
+class SyncController extends Controller {
+    public function sync() {
+
+        $tokenCache = new TokenCache();
+        $accessToken = $tokenCache->getAccessToken();
+
+        $graph = new Graph();
```

Comment:
can this be configured as service?? the usage may be easier along the app.

Reply:
good idea -> future enhancements